docs/blueprints/deletion-performance.md

```diff
@@ -0,0 +1,129 @@
+</tr>
+</thead>
+<tbody>
+<tr style="border: 0.1em solid grey"><td style="border: 0.1em solid grey">1 Kib</td><td style="border: 0.1em solid grey">yes</td><td style="border: 0.1em solid grey">858</td><td style="border: 0.1em solid grey">277</td><td style="border: 0.1em solid grey">272</td><td style="border: 0.1em solid grey">144</td><td style="border: 0.1em solid grey">140</td><td style="border: 0.1em solid grey">0</td></tr><tr style="border: 0.1em solid grey"><td style="border: 0.1em solid grey">4 Kib</td><td style="border: 0.1em solid grey">yes</td><td style="border: 0.1em solid grey">910</td><td style="border: 0.1em solid grey">293</td><td style="border: 0.1em solid grey">278</td><td style="border: 0.1em solid grey">144</td><td style="border: 0.1em solid grey">142</td><td style="border: 0.1em solid grey">0</td></tr><tr style="border: 0.1em solid grey"><td style="border: 0.1em solid grey">5 Kib</td><td style="border: 0.1em solid grey">no</td><td style="border: 0.1em solid grey">1959</td><td style="border: 0.1em solid grey">328</td><td style="border: 0.1em solid grey">275</td><td style="border: 0.1em solid grey">142</td><td style="border: 0.1em solid grey">513</td><td style="border: 0.1em solid grey">652</td></tr><tr style="border: 0.1em solid grey"><td style="border: 0.1em solid grey">10 Kib</td><td style="border: 0.1em solid grey">no</td><td style="border: 0.1em solid grey">2451</td><td style="border: 0.1em solid grey">308</td><td style="border: 0.1em solid grey">278</td><td style="border: 0.1em solid grey">141</td><td style="border: 0.1em solid grey">560</td><td style="border: 0.1em solid grey">1134</td></tr><tr style="border: 0.1em solid grey"><td style="border: 0.1em solid grey">2.2 Mib</td><td style="border: 0.1em solid grey">no</td><td style="border: 0.1em solid grey">2643</td><td style="border: 0.1em solid grey">325</td><td style="border: 0.1em solid grey">285</td><td style="border: 0.1em solid grey">149</td><td style="border: 0.1em solid grey">560</td><td style="border: 0.1em solid grey">1273</td></tr><tr style="border: 0.1em solid grey"><td style="border: 0.1em solid grey">256 Mib</td><td style="border: 0.1em solid grey">no</td><td style="border: 0.1em solid grey">7591</td><td style="border: 0.1em solid grey">333</td><td style="border: 0.1em solid grey">275</td><td style="border: 0.1em solid grey">145</td><td style="border: 0.1em solid grey">1539</td><td style="border: 0.1em solid grey">6644</td></tr>
+</tbody>
+</table>
+
+We extracted the data of the table the following trace graph files:
+
+* [1kib](images/deletion-perfomance-1.0kb-file-trace.svg)
+* [4Kib](images/deletion-perfomance-4.0kb-file-trace.svg)
+* [5Kib](images/deletion-perfomance-5.0kb-file-trace.svg)
+* [10kib](images/deletion-perfomance-10.0kb-file-trace.svg)
+* [2.2Mib](images/deletion-perfomance-2.2mb-file-trace.svg)
+* [256Mib](images/deletion-perfomance-256mb-file-trace.svg)
```

Comment:
Do we need to commit these svg files, they are kind of large.

Reply:
Maybe not before merging to master, but I think that they were good to share at this stage because they provide the source of the data in the added table and reviewers can have a look to check that I haven't misinterpreted them.

In my opinion, we should keep them, if not in the repo, in another public place, because they provide transparency to the community once we publish this design doc in the forum.

Reply:
We could use our linksharing. Also gists support uploading svgs.

Reply:
I will move them to a gists, right now it's the straightforward alternative.